tomviz/pybind11/ctvlib.h

```diff
@@ -0,0 +1,84 @@
+
+//
+//  ctvlib.hpp
+//
+//
+//  Created by Hovden Group on 5/6/19.
```

Comment:
This cannot be merged until this line is removed, and the standard C++ copyright header is inserted,
`/* This source file is part of the Tomviz project, https://tomviz.org/.
   It is released under the 3-Clause BSD License, see "LICENSE". */`
All C++ is BSD licensed, attribution is through Git history, and we cannot have lines with "All Rights Reserved" in them as they are not open source.